src/main/java/core/basesyntax/service/GenerateStringToWrite.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import java.util.Map;
+
+public interface GenerateStringToWrite {
```

Comment:
All services should be nouns and contain the word "Service" in their names 
Also, naming should be understandable for someone who could see your code for the first time. What does that mean `GenerateStringToWrite`? Better to name sth like "ReportMakerService", "ReportGeneratorService" etc.

Fix naming in all services and their implementations according to the naming convention.